.github/workflows/ci-pipeline.yml

```diff
@@ -38,6 +43,16 @@ jobs:
       run: |
         touch .env
         echo "${{ secrets.SERVER_ENV }}" > .env
+    
+    - name: Terraform
+      run: | 
+        terraform init && \
+        terraform apply \
+          --auto-approve \
+          -var="oauth_app_id=${{ secrets.OAUTH_APP_ID }}" \
+          -var="oauth_app_secret=${{ secrets.OAUTH_APP_SECRET }}" \
+          -var="secret_key=${{ secrets.SECRET_KEY }}" \
+          -var="env=production" \
```

Comment:
I think you've missed updating the final stage to trigger a release on Azure - so if you just update the Docker image your Terraform won't be updated; e.g. your "Trigger Azure deployment" step [in this pipeline](https://github.com/TomJSoftwire/Devops-Course-Starter/actions/runs/3097472354/jobs/5014187070) says it released to https://thojoh-module-9.scm.azurewebsites.net - we want to update that to the `curl -dH -X POST "$(terraform output -raw cd_webhook)"` command you hopefully tested out locally